hub/integrations/__init__.py

```diff
@@ -1,7 +1,14 @@
+from hub.util.exceptions import WindowsSharedMemoryError
+
+
 try:
+    import os
+
+    if os.name == "nt":
+        raise WindowsSharedMemoryError
```

Comment:
does this mean that windows contributors cannot develop for our pytorch integration?

Reply:
this is caught just below, so people won't see it raised